### DISCUSSION TITLE
change time for event view

### DISCUSSION THREAD

**Discussion by Synt3x** [blakeblackshear/frigate#13672] [Ask A Question] [answered]
Hello,

Is it possible to change eventview from am/pm to 24h clock? Couldnt find this menu.
![image](https://github.com/user-attachments/assets/d386d6f7-2c6e-417f-a364-5eb7f4c87b7d)

/Robin

**Comment by hawkeye217** [comment, marked as answer]
See the [FAQ](https://github.com/blakeblackshear/frigate/discussions/12724), Usage question 13.
